app/controllers/events_controller.rb

```diff
@@ -250,6 +251,37 @@ def destroy
     redirect_to events_path
   end
 
+  def copy_event_and_set_insisibiliy
```

Comment:
here is a typo, right? should be ...invisibility

Reply:
<f1a50fc4e0188a4b8b2c8f43a39df182d4ad6a82>